src/logger/errorHandling.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response, NextFunction } from 'express';
+
+export const errorHandling = (err: Error, req: Request, res: Response, next: NextFunction) => {
+    if (res.headersSent) {
+        return next(err);
+    }
+    res.status(500);
+    res.render('error', { error: err });
```

Comment:
Did you test it? Fails with
> <pre>Error: No default engine was specified and no extension was provided.<br> &nbsp; &nbsp;at new View (/source/node_modules/express/lib/view.js:61:11)<br> &nbsp; &nbsp;at Function.render (/source/node_modules/express/lib/application.js:587:12)<br> &nbsp; &nbsp;at ServerResponse.render (/source/node_modules/express/lib/response.js:1039:7)<br> &nbsp; &nbsp;at errorHandling (/source/src/logger/errorHandling.ts:8:9)<br> &nbsp; &nbsp;at Layer.handle_error (/source/node_modules/express/lib/router/layer.js:71:5)<br> &nbsp; &nbsp;at trim_prefix (/source/node_modules/express/lib/router/index.js:326:13)<br> &nbsp; &nbsp;at /source/node_modules/express/lib/router/index.js:286:9<br> &nbsp; &nbsp;at Function.process_params (/source/node_modules/express/lib/router/index.js:346:12)<br> &nbsp; &nbsp;at next (/source/node_modules/express/lib/router/index.js:280:10)<br> &nbsp; &nbsp;at Layer.handle_error (/source/node_modules/express/lib/router/layer.js:67:12)</pre>

Reply:
`res.sendStatus` should be enough. Usually we don't want to disclose the internal error to the outside world

Reply:
Done, thank you